src/test/java/org/folio/rest/impl/NoteTypesImplTest.java

```diff
@@ -536,7 +534,7 @@ public void shouldReturn400WhenUserIdIsMissing() {
   }
 
   @Test
-  public void shouldReturn401WhenIncorrectTenant() {
+  public void shouldReturn400WhenIncorrectTenant() {
```

Comment:
Similarly, when incorrect tenant is given, why did we use unauthorized before and now a bad request?

Reply:
@sduvvuri-ebsco thanks, fixed